### ISSUE TITLE
Time Picker

### ISSUE THREAD

**Issue by modernkd** [developit/preact-compat-example#10]
<img width="312" alt="screen shot 2017-02-04 at 7 39 15 pm" src="https://cloud.githubusercontent.com/assets/4150097/22622744/a84f6572-eb11-11e6-8d24-b5f3a4349bdb.png">

After choosing the hour, it tries writing the minutes over the hours and if you try choosing, it overwrites the hour choice.

**Comment by developit** [comment]
yup yup. not sure what's causing this. Which versions of preact and preact-compat are you checking this with?

**Comment by modernkd** [comment]
I was just going through the link you had posted. I have no idea why it is messing up there

**Comment by developit** [comment]
It's using an older version of Preact and preact-compat. I need to update the demo to check.if it's fixed in the latest versions.
